### DISCUSSION TITLE
Pricing: is it paid or free?

### DISCUSSION THREAD

**Discussion by lbineau** [AdguardTeam/AdGuardHome#6532] [General]
Hello dear community,

I've read and watched a lot of videos about PiHole vs Blocky vs AdGuard Home and none of them mention the pricing of AdGuard. 
I have to say I'm a bit confused.
Is Ad Guard different from Ad Guard Home? https://adguard.com/en/license.html
Is the project open source and free to use on our server (raspberry for instance)?

It isn't very obvious as there is a big PURCHASE button on the AdGuard Home page.
<img width="1267" alt="screenshot of AdGuard Home page showing a purchase button" src="https://github.com/AdguardTeam/AdGuardHome/assets/1534688/1dffff36-c109-447e-823e-b83c6f1babe9">

Thank you :-)

**Comment by ppfeufer** [comment]
AdGuardHome is opensource and free.

**Reply by lbineau** [reply]
Thank you for the answer, this was not obvious for me.
